mod_bot/conf.py

```diff
@@ -0,0 +1,81 @@
+import os
+import json
+import pprint
+
+def read_config(path):
+    print("reading configuration")
+    if os.path.exists(path) is False:
+        return {}
+
+    with open(path, 'r') as f:
+        conf = json.load(f)
+    load_guilds_config(conf)
+    return conf
+
+def load_guilds_config(conf):
+    if "filepath" in conf and "listen_to" in conf:
+        for gid in list(conf["listen_to"]):
```

Comment:
I don't like abbreviated name because you have to think to read them, code should be more fluid

Reply:
Given the context ("load_guilds_config"), I think the name is explicit enough